bin/tictactoe

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Global Thermonuclear War!"
+sleep(3)
+puts "Apologies, we are having technical difficulties. Let's play Tic-Tac-Toe in the meantime."
+sleep(3)
+puts "Welcome to Tic-Tac-Toe!"
+
+Game.start
```

Comment:
Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.